src/w3c/templates/headers.js

```diff
@@ -274,6 +275,58 @@ export default (conf, options) => {
   </div>`;
 };
 
+function renderFeedback(conf) {
+  if (!conf.github && !conf.wgPublicList) return;
+  const definitions = [];
+
+  // Github feedback...
+  if (conf.github) {
+    const { repoURL, issuesURL, newIssuesURL, pullsURL, fullName } =
+      conf.github;
+    definitions.push(
+      html`<dd>
+        <a href="${repoURL}">GitHub ${fullName}</a>
+        (<a href="${pullsURL}">pull requests</a>,
+        <a href="${newIssuesURL}">new issue</a>,
+        <a href="${issuesURL}">open issues</a>)
+      </dd>`
+    );
+  }
+
+  if (conf.wgPublicList) {
+    // The <a href="mailto:list?subject"> link for the public list
```

Comment:
Nit: With this, if we collapse the `if {}` in text editor, the comment identify the block.
```suggestion
  // The <a href="mailto:list?subject"> link for the public list
  if (conf.wgPublicList) {
```